celeri/get_mesh_edge_elements.py

```diff
@@ -0,0 +1,56 @@
+def get_mesh_edge_elements(meshes):
+    # Find indices of elements lining top, bottom, and sides of each mesh
+    
+	from ismember import ismember
```

Comment:
@jploveless I'm not able to do this import "from ismember import ismember".  I don't think it's part of the celeri environment.yml.  I'm wondering if there's something built into numpy that might do this.  Perhaps something in the discussion here: https://stackoverflow.com/questions/25923027/matlab-ismember-function-in-python

Reply:
I tried some of those (or from another SO post) but the the [ismember package](https://pypi.org/project/ismember/) seemed to be more straightforward in comparing whole rows and just returning row indices, which was important.

Reply:
Understood and I just added ismember as a pip import to environment.yml!